composer.json

```diff
@@ -19,16 +19,17 @@
         "php": ">=7.3",
```

Comment:
[error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Unable to install the <em class="placeholder">admin_toolbar</em> module since it does not exist.
Unable to install the <em class="placeholder">admin_toolbar_tools</em> module since it does not exist.
Unable to install the <em class="placeholder">blazy</em> module since it does not exist.
Unable to install the <em class="placeholder">slick</em> module since it does not exist.
Unable to install the <em class="placeholder">bootstrap</em> theme since it does not exist. in Drupal\Core\Config\ConfigImporter->validate() (line 748 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php).